lib/cipherstash/index/bloom_filter.rb

```diff
@@ -1,28 +1,115 @@
-require "openssl"
```

Comment:
How is `OpenSSL` getting loaded if you're not loading it here?  (presumably some other part of the code you're running before this is also doing a `require "openssl"`, but relying on that *always* being the case is a recipe for the most frustratingly obtuse support tickets).